satellite/satellitedb/overlaycache.go

```diff
@@ -565,10 +565,19 @@ func (cache *overlaycache) CreateStats(ctx context.Context, nodeID storj.NodeID,
 		return nil, Error.Wrap(errs.Combine(err, tx.Rollback()))
 	}
 
-	if startingStats == nil {
-		// TODO: add possible sanity checking for alpha and beta values
-
-		startingStats = &overlay.NodeStats{}
+	if startingStats != nil {
+		if startingStats.AuditReputationAlpha <= 0 {
```

Comment:
I would do these checks before starting the transaction

Reply:
it turns out we never used this function, oops